### DISCUSSION TITLE
Insufficient occupation of GPU

### DISCUSSION THREAD

**Discussion by C-BJ** [bevyengine/bevy#4415] [Q&A]
### My computer:
CPU: Intel I710710U
GPU: Intel UHD 620
RAM: 16.0 GiB
System: Windows11 22000.556
**-----------------------------------**

I run the official use case of bevy on my computer, but no matter how many FPS (20 or 60), GPU accounts for only 60% - 80%

I'm sure there are no system memory leaks

Running modern games and game engine GPU on my computer will be full load

However, I noticed that one of the 12 CPU cores will run full no matter which official example is running, and the other core will almost run full (the two cores are the same in each run)

**Comment by C-BJ** [comment]
When I start running the program, there will be a white rectangle and a black outer ring in the window for about 1 second. In any official example, including the UI example

**Comment by bjorn3** [comment]
Have you compiled the examples in release mode? You can pass `--release` to cargo to do that.

**Comment by C-BJ** [comment]
**I don't know why, but it works now**
But Why when I start running the program, there will be a white rectangle and a black outer ring in the window for about 1 second. In any official example, including the UI example?
